### PR TITLE
150 integrate blue delta stats with backend

### PR DESCRIPTION
**Pull request by smebrahimi**
This issue #150

### CODE REVIEW

**Comment by paulchrisluke** [comment]
can you please link the blue delta stats button here https://deploy-preview-299--eloquent-thompson-534232.netlify.app/stats

to the page you built so we can test? thank you!

**Comment by smebrahimi** [comment]
> can you please link the blue delta stats button here https://deploy-preview-299--eloquent-thompson-534232.netlify.app/stats
> 
> to the page you built so we can test? thank you!

**Comment by smebrahimi** [comment]
@paulchrisluke Done! Your welcome!

File: src/store/modules/email.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { JumpStartService } from '@whynotearth/meredith-axios';
-import qs from 'qs';
+import { BASE_API } from '@/connection/api.js';
 import { debounce } from 'lodash-es';
+import store from '@/store';
```
Comment (abdullahcalisir12) [inline src/store/modules/email.js:4]: store is not used in the file so no need to import

**Comment by abdullahcalisir12** [comment]
image

And here at top box-shadow is missing

**Comment by paulchrisluke** [comment]
@mort3za @smebrahimi please merge this to staging when you can if its ready. this is a feature needed for tonight

File: src/providers/ActivityListItem.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  </div>
+</template>
+
+<script>
+import BaseChip from '@/components/BaseChip.vue';
+import { formatDate } from '@/helpers.js';
+
+export default {
+  name: 'ActivityListItem',
+  props: ['model'],
```
Comment (atharva3010) [inline src/providers/ActivityListItem.vue:51]: We can declare the prop in object syntax here too.

File: src/providers/ActivityList.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <NavigationBottom />
+    </template>
+  </LayoutFixedScrollable>
+</template>
+
+<script>
+import BaseAppBarHeader from '@/components/BaseAppBarHeader.vue';
+import ActivityListItem from './ActivityListItem.vue';
+import LayoutFixedScrollable from '@/components/LayoutFixedScrollable.vue';
+import { mapGetters, mapActions } from 'vuex';
```
Comment (atharva3010) [inline src/providers/ActivityList.vue:31]: These are imported by not used

File: src/providers/ActivityFeedDetail.vue [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+<script>
+import BaseAppBarHeader from '@/components/BaseAppBarHeader.vue';
+import BaseTabs from '@/components/BaseTabs.vue';
+import BaseTab from '@/components/BaseTab.vue';
+import ActivityListItem from './ActivityListItem.vue';
+import ActivityFeedReadReportLog from '@/components/ActivityFeedReadReportLog.vue';
+import LayoutFixedScrollable from '@/components/LayoutFixedScrollable.vue';
+import NavigationBottom from '@/components/BaseNavigationBottom';
+import { get } from 'lodash-es';
+import { formatDate } from '@/helpers.js';
```
Comment (atharva3010) [inline src/providers/ActivityFeedDetail.vue:56]: It's okay using it this time, but we should only import external functions for complex stuff, in your particular use-case, it's doesn't add many benefits.

**Pull request merged**